MMA Removal Utility/Prerequisites.md

```diff
@@ -0,0 +1,67 @@
+## **Prerequisites**
+
+### **Validate prerequisites on machine**  
+<br/>
+
+  1.  Installation steps are supported using following OS options: 	
+
+      - Windows 10 or higher
+      - Windows Server 2019 or higher
+  
+  <br/>
+
+  2. PowerShell 5.0 or higher:
+
+      All setup steps will be performed with the help of PowerShell ISE console. If you are unaware of PowerShell ISE, refer [link](PowerShellTips.md) to get a basic understanding.
+      Ensure that you are using Windows OS and have PowerShell version 5.0 or higher by typing **$PSVersionTable** in the PowerShell ISE console window and looking at the PSVersion in the output as shown below.) 
```

Comment:
Ensure that you are using Windows OS and have PowerShell version 5.0 or higher -> Validate that you have PowerShell version 5.0 or higher

Reply:
Addressed.